content/working_with/console/advanced-configuration.md

```diff
@@ -9,10 +9,11 @@ weight: 170
 
 You can configure more advanced customization of Cloudify Console by modifing user configuration file residing on Cloudify Manager machine.
 
-Configuration file path is `/opt/cloudify-stage/conf/userConfig.json`. You can see a list of available settings and default values in [cloudify-cosmo/cloudify-stage repository](https://github.com/cloudify-cosmo/cloudify-stage/blob/master/conf/userConfig.json).
+To do that:
+1. Copy `/opt/cloudify-stage/conf/userConfig.json` file to `/opt/cloudify-stage/dist/userData/userConfig.json`.
+1. Apply modifications (optionally remove all not modified parts). 
+1. Restart Cloudify Console service on Cloudify Manager machine by executing: `sudo service cloudify-stage.service restart`.
 
-Please be aware that for changes to make effect, you have to restart Cloudify Console service on Cloudify Manager machine by executing:
-
-```
-service cloudify-stage.service restart 
-```
+Check out the following links to get more information:
+ * [User configuration file format](https://github.com/cloudify-cosmo/cloudify-stage/blob/master/conf/userConfig.json)
+ * [User configuration file description](https://github.com/cloudify-cosmo/cloudify-stage/blob/master/conf/README.md#user-userconfigjson) 
```

Comment:
It seems this file doesn't render correctly:
![obraz](https://user-images.githubusercontent.com/49753069/65775039-af322880-e13f-11e9-9e57-e89642caa723.png)

Please see https://docs.cloudify.co/staging/CY-1736/working_with/console/advanced-configuration/

Reply:
Fixed.